corehq/apps/performance_sms/_design/views/by_domain/map.js

```diff
@@ -0,0 +1,5 @@
+function (doc) {
+    if(doc.doc_type === "PerformanceConfiguration") {
+        emit(doc.domain, null);
+    }
+}
```

Comment:
This is a fair question. I still find couch to be nice to work with which is the main reason. As to why I don't think it's a problem for them to be in couch:

- These models are going to be at most a handful per domain and rarely saved so won't contribute to a heavy view indexing load
- They are being saved to the `meta` DB, so these views will be super fast to reindex and only run over a very small amount of documents. They also won't contribute to the main DB indexing load.

We could easily port to SQL though it doesn't seem obviously worth it to me. Open to push back on that though.